source/_integrations/xiaomi_miio.markdown

```diff
@@ -216,6 +216,8 @@ Supported devices:
 | Air Purifier 3 (2019)  | zhimi.airpurifier.ma4  | |
 | Air Purifier 3H (2019) | zhimi.airpurifier.mb3  | |
 | Air Purifier 3C        | zhimi.airpurifier.mb4  | |
+| Air Purifier 4         | zhimi.airp.mb5         | |
+| Air Purifier 4 PRO     | zhimi.airp.vb4         | |
```

Comment:
Why are you using `zhimi.airp.mb5` here instead of `zhimi.airpurifier.mb5`?

Reply:
that is the identifier for these 2 models and it is according to the python-miio library(https://github.com/rytilahti/python-miio/blob/3c6b8dca6bc799e2d8df8f506e26fae3d5f424e4/miio/integrations/airpurifier/zhimi/airpurifier_miot.py#L245)

Reply:
If so, I think those model names should also be used on line 501.

Reply:
thanks, i updated it.